CHANGELOG.md

```diff
@@ -6,6 +6,76 @@ in this file.
 Detailed changes to the Solar Forecast Arbiter Core python library can be found
 in the core documentation's [what's new](https://solarforecastarbiter-core.readthedocs.io/en/latest/whatsnew/index.html) series.
 
+
+## [1.0beta5] - 2020-04-27
+
+### Added
+
+- Added support for event forecasts and associated reports. A list of supported
+  metrics can be found on the [metrics page](https://solarforecastarbiter.org/metrics/#metrics-for-deterministic-event-forecasts).
+
+- Reports now support normalized metrics such as NMAE. Normalization is
+  automatically determined based on Observation type. AC power observations
+  are normalized by AC capacity; DC power by DC capacity. Normalized metrics
+  are set to nan for all other variables.
+
+- Users may now select a reference forecast to use in computing
+  [forecast skill](https://solarforecastarbiter.org/metrics/#s).
+
+- Many deterministic metrics may now include a deadband to account for
+  uncertainty. The deadband is specified as a percentage of the observations.
+  The error forecast - observation is set to 0 within the deadband. Users may
+  control the deadband using the report creation form.
+
+- Reports of GHI observations and forecasts may now filter out periods of
+  clear sky. Users must generate two reports if they desire metrics for both
+  all sky and cloudy sky.
+
+- All report plots are now created with Plotly instead of Bokeh. Plotly's JSON
+  specification and more mature javascript library makes it easier to optimize
+  reports for display in web browsers.
+
+- Added API endpoint to recompute report at `/reports/<report_id>/recompute`.
+  API users can access this endpoint now, and a button will be added to the
+  dashboard in a future release.
+
+- Added API endpoint to get the actions the current user is allowed to take on
+  an object at `/users/actions-on/<object_id>`.
+
+- Added API endpoints to query for latest available data or a timerange of
+  available data. These endpoints are found by appending `/latest` or
+  `/timerange` to the end of a values endpoint.
+
+- Added DOE ARM sites to reference database.
+
+### Changed
+
+- Individual forecasts or observations can be singled out by double-clicking
+  the legend of a report timeseries or scatter plot.
+
+- Dashboard hides buttons for actions that a users cannot take due to
+  insufficient permissions. This feature has been implemented on individual
+  site, forecast, probabilistic forecast, aggregate, and observation pages.
+  This feature will be extended to tabulated listing pages in a future release.
```

Comment:
@wholmgren @alorenzo175 Does this clarify things? Should we have the note about extending this to other pages or leave it off?

Reply:
It seems to still not quite describe the situation. Doesn't it really just apply to delete?

Reply:
No, it extends to uploading and downloading data, adding observations to an aggregate, adding permissions to a role, and granting or revoking a role. Would it just be simpler if I created a list of these here?

Reply:
maybe it's ok to leave the existing text if you mention the future extention also applies to the create permissions